pkg/node/sendoperation/readonlycall.go

```diff
@@ -36,7 +36,7 @@ func EstimateGasCostCallSC(
 
 	estimatedGasCost := uint64(result.GasCost)
 
-	return estimatedGasCost, nil
+	return estimatedGasCost + DefaultInstalCreationCost, nil
```

Comment:
warning here, the value can go beyond the maximum value allowed for max gas, we should do something like max(estimatedGasCost + DefaultInstalCreationCost, MaxGasAllowedCallSC)